src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final int LINE_WIDTH = 60;
+
+    //to draw a straight line
+    private static void printHorizontalLine() {
+        System.out.println("_".repeat(LINE_WIDTH));
+    }
+
+    public static String[] taskLister(String[] todos) {
+
+        int todoCount = 0;
+        String[] result = new String[100];
+        for (String todo : todos) {
+            //System.out.println(word);
+            if (todo != null) {
+                result[todoCount] = todo;
+                todoCount++;
+            } else {
+                break;
+
+            }
+        }
+        return Arrays.copyOf(result, todoCount);
+    }
+
+    public static boolean splitString(String line) {
```

Comment:
Should this sound more boolean? e.g. isMatched, isString etc etc

Reply:
Perhaps try not to do too many things in one method like splitting the string and checking if it's boolean in the same method.